controllers/medusa/backup_controller_test.go

```diff
@@ -115,7 +115,7 @@ func testBackupDatacenter(t *testing.T, ctx context.Context, f *framework.Framew
 		}
 
 		condition := findDatacenterCondition(k8ssandraStatus.Cassandra, cassdcapi.DatacenterScalingUp)
-		return !(condition == nil && condition.Status == corev1.ConditionFalse)
+		return !(condition != nil && condition.Status == corev1.ConditionFalse)
```

Comment:
I find this really hard to grasp. Couldn't we simplify to:
```suggestion
		return condition == nil || condition.Status != corev1.ConditionFalse
```